packages/suite-native-settings/src/screens/SettingsScreen.tsx

```diff
@@ -1,37 +1,40 @@
 import React from 'react';
-import { Box, Button, Text } from '@suite-native/atoms';
-import { View } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { SettingsStackRoutes, SettingsStackParamList } from '../navigation/routes';
 import { StackProps } from '@suite-native/navigation';
+import { VStack } from '@suite-native/atoms';
+
+import { ApplicationSettings } from '../components/ApplicationSettings';
+import { DeviceSettings } from '../components/DeviceSettings';
+import { CoinsSettings } from '../components/CoinsSettings';
+import { DeviceActionButtons } from '../components/DeviceActionButtons';
+import { SettingsStackParamList, SettingsStackRoutes } from '../navigation/routes';
 
-const settingsScreenStyle = prepareNativeStyle(() => ({
+const settingsSectionsStyle = prepareNativeStyle(utils => ({
     flex: 1,
     justifyContent: 'center',
-    alignItems: 'center',
+    padding: utils.spacings.medium,
 }));
 
 export const SettingsScreen = ({
     navigation,
 }: StackProps<SettingsStackParamList, SettingsStackRoutes.Settings>) => {
     const { applyStyle } = useNativeStyles();
 
+    const handleRedirect = (route: SettingsStackRoutes | undefined) => {
+        if (!route) return;
```

Comment:
Didn't we agree on not using one line ifs? I am honestly not sure :D

Reply:
I thought we agreed if it's just one line condition for these kind of checks it's fine. The debate was more about the curly braces if there is else clause if I remember correctly. Am I wrong?

Reply:
Perhaps you are right. I am just not a fan of one line ifs at all, wether it's with else, with some constant assignment or a return. It's much clearer with the braces and an additional tab but that's just my opinion.